### PR TITLE
Add support for priority lock

### PR DESCRIPTION
**Pull request by tetienne**
Screenshot 2021-08-24 at 09-34-26 Configuration - Home Assistant
Screenshot 2021-08-24 at 09-33-02 Configuration - Home Assistant

### CODE REVIEW

File: custom_components/tahoma/sensor.py [hunk 1]
```diff
@@ -289,6 +285,21 @@
...
+    OverkizSensorDescription(
+        key="io:PriorityLockOriginatorState",
+        name="Priority Lock Originator",
+        value=lambda value: str(value).capitalize(),
+        icon="mdi:lock",
+        entity_registry_enabled_default=False,
+    ),
+    OverkizSensorDescription(
+        key="core:PriorityLockTimerState",
+        value=lambda value: value,
```
Comment (iMicknl) [inline custom_components/tahoma/sensor.py:297]: You can remove this, if you don't want to do any transformation.

File: custom_components/tahoma/sensor.py [hunk 2]
```diff
@@ -289,6 +285,21 @@
         value=lambda value: str(value).capitalize(),
         entity_registry_enabled_default=False,
     ),
+    OverkizSensorDescription(
+        key="io:PriorityLockOriginatorState",
+        name="Priority Lock Originator",
+        value=lambda value: str(value).capitalize(),
```
Comment (iMicknl) [inline custom_components/tahoma/sensor.py:291]: What will be the default value, if there is nothing? Will str(value) be 'none'?

Comment (tetienne) [inline custom_components/tahoma/sensor.py:291]: I hesitate with *Unlocked* as default value for this one, instead of *Unknown*.

Comment (iMicknl) [reply custom_components/tahoma/sensor.py:291]: Nevermind, it seems that this is catched below :-)

Comment (tetienne) [reply custom_components/tahoma/sensor.py:291]: Look at my screenshot :)

Comment (tetienne) [reply custom_components/tahoma/sensor.py:291]: Look at my screenshot :)

Comment (iMicknl) [reply custom_components/tahoma/sensor.py:291]: Which screenshot? ;-)

Comment (tetienne) [reply custom_components/tahoma/sensor.py:291]: Damned, they were not updated. Should be ok now.

**Review by iMicknl — APPROVED** [review]

File: custom_components/tahoma/cover_devices/tahoma_cover.py [hunk 3]
```diff
@@ -128,19 +123,6 @@ def is_closed(self):
 
         return None
 
-    @property
-    def icon(self):
```
Comment (iMicknl) [inline custom_components/tahoma/cover_devices/tahoma_cover.py:132]: Finally we got rid of this part <3

Comment (tetienne) [reply custom_components/tahoma/cover_devices/tahoma_cover.py:132]: 💪

**Pull request merged**